inventory_management/inventory_management/doctype/stock_entry/stock_entry.json

```diff
@@ -0,0 +1,85 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2024-01-11 17:22:55.543930",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
```

Comment:
This form design still has problems:

- The child table should move to a separate section and span all columns since that's the main focus area of the form.
- Form labels should always be capitalized -> **Stock Entry Items** or you could just label it **Items** because its already inside the Stock Entry form
- Current Date & Time should be fetched automatically
- Based on my type, my source and/or target warehouses should become mandatory so I know what to fill up beforehand and I don't need to hit save and wait for it. Handle this in your client-side code.

<img width="1292" alt="image" src="https://github.com/vinyselopal/InventoryManagementFrappe/assets/24353136/ba399339-54eb-4040-9498-033be337ed5f">